airflow/www/views.py

```diff
@@ -2291,6 +2294,32 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, LoggingMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except Exception as e:
+            airflow_version = None
+            self.logger.error(e)
+
+        # Get the Git repo and git hash
+        git_version = None
+        try:
+            with open("airflow/git_version") as f:
```

Comment:
Done, though I am using os.path.join(*[settings.AIRFLOW_HOME, 'airflow', 'git_version']) here and in setup.py